server/market/market_test.go

```diff
@@ -647,7 +647,8 @@ func TestMarket_Run(t *testing.T) {
 	auth.handlePreimageDone = make(chan struct{}, 1)
 
 	ctx, cancel := context.WithCancel(context.Background())
-	startEpochIdx := 1 + encode.UnixMilli(time.Now())/epochDurationMSec
+	defer cancel()
+	startEpochIdx := 2 + encode.UnixMilli(time.Now())/epochDurationMSec
```

Comment:
The first part of this test is submitting and order to a market that hasn't hit it's start epoch yet, and if the code between here and [SubmitOrder here](https://github.com/decred/dcrdex/pull/427/files#diff-92957aac6dd698f2d846e3b99ed7281dR728) takes too long, the market will be running and the expected error won't be hit.  So 1 -> 2 for some extra time.